lib/routes/cdirt/namespace.ts

```diff
@@ -0,0 +1,10 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: '成都轨道交通职业学院',
+    url: 'edua.chengdurail.com',
+    description: `:::
+    成都轨道交通职业学院，是成都轨道交通集团举办的全日制高等学校。尚未规定英文缩写，暂以 Chengdu Institute of Railway Transit 的首字母为依据。
+    :::`,
```

Comment:
```suggestion
成都轨道交通职业学院，是成都轨道交通集团举办的全日制高等学校。尚未规定英文缩写，暂以 Chengdu Institute of Railway Transit 的首字母为依据。
:::`,
```
Indent the content with either 0 or 2 spaces or the markdown content won't be rendered properly.